ocaml/xapi/xapi_sr.ml

```diff
@@ -778,13 +783,40 @@ let scan ~__context ~sr =
               Db.VDI.get_records_where ~__context
                 ~expr:(Eq (Field "SR", Literal sr'))
             in
+            let string_of_l lst =
+              List.map (fun (_, v) -> v.vDI_uuid) lst |> String.concat ","
```

Comment:
We could simply report the Refs (not the UUIDs); and these could be taken simply from the sets. While the API operates on Refs, I would agree that UUIDs often have more meaning for a user.